src/windows/top_toolbar.c

```diff
@@ -1461,10 +1462,11 @@ static void window_top_toolbar_scenery_tool_down(short x, short y, rct_window *w
 	case SCENERY_TYPE_SMALL:
 	{
 		int quantity = 1;
-		if (window_scenery_is_build_cluster_tool_on) {
-			quantity = 35;
+		bool isCluster = window_scenery_is_build_cluster_tool_on && network_can_perform_action(network_get_current_player_group_index(), GAME_COMMAND_TOGGLE_SCENERY_CLUSTER);
+		if (isCluster) {
+			quantity = 36;
```

Comment:
@janisozaur  but to be real though, i did it because i have 2 forks of this project. one for experimenting with, and two for the push i did of this. I could't figure out ho to get 2 forks of the code without having to spend alot of time and effort. 

@IntelOrca also, i'm looking at this code of the chat command, and honestly it's not making much sense to me on how it works. it's doing a "packet << (packet)chat_command;" (or something along that line), and it's just, i'm gonna have to study how this chat command works which would take some time. and honestly right now... sure it's a little hacky but hey, the code works. plus openrct2 is not even in an alpha state. i can switch this up later when i get time but im just busy with alot of other stuff as of the last few days, and my current goal openrct2-relatd is figuring out why a window i made is skipping a widget index in mousedown (it's reading me pressing widget 4 when i'm clicking over widget 3, but likely i did something wrong in the code that i just don't see) 

i'm sure right now the multiplayer community would care more about getting a feature that might not be properly implemented right under the hood than not get it at all, right? it's like indie game developing, in the end the person playing the game doesn't care if you use unity, gamemaker, or unreal, if it's a good game they'll like it. 

if you want to take this idea and adjust it to how silencer suggested if you think it would be easy for you, go ahead and take my work and recreate it yourself, you don't even need to credit me :)